File: Fase 2/ServidorRevistas/src/main/java/objetos/CostoHosting.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package objetos;

import java.util.Date;

/**
 *
 * @author fernanrod
 */
public class CostoHosting {
    private Date fechaInicio;
    private Float porcentaje;

    public CostoHosting(Date fechaInicio, Float porcentaje) {
        this.fechaInicio = fechaInicio;
        this.porcentaje = porcentaje;
    }
    public Date getFechaInicio() {
        return fechaInicio;
    }
    public Float getPorcentaje() {
        return porcentaje;
    }
}

File: Fase 2/revistas-app/src/app/objetos/base/CostoHosting.ts
export class CostoHosting {
    private _fechaInicio: Date;
    private _porcentaje: number;
    
    constructor(fechaInicio: Date, porcentaje: number) {
        this._fechaInicio = fechaInicio;
        this._porcentaje = porcentaje;
    }
    
    public get fechaInicio() : Date {
        return this._fechaInicio;
    }
    public get porcentaje() : number {
        return this._porcentaje;
    }
}